src/gui_common/ChemicalEquation.cs

```diff
@@ -182,7 +182,10 @@ public override void _Process(double delta)
         }
 
         if (AutoRefreshProcess)
+        {
+            AutoRefreshProcess = false;
```

Comment:
As this is static this will cause only a *single* chemical equation to refresh which will then set this to false and not allow any of the other instances to see this variable being true.